.eslintrc.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
+    env: {
+        browser: true,
+        es6: true,
+        node: true,
+        jest: true,
+    },
+    extends: [
+        'plugin:react/recommended',
+        'plugin:jest/recommended',
+        'plugin:react-hooks/recommended',
+        'plugin:storybook/recommended',
+        'prettier',
+    ],
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        project: 'tsconfig.eslint.json',
+    },
+    plugins: ['@typescript-eslint', 'eslint-comments', 'jest', 'storybook'],
+    root: true,
+    rules: {
```

Comment:
I'd keep everything by default from plugins unless it is essential to have them here with custom setup